modules/python/cython/test_des.py

```diff
@@ -0,0 +1,30 @@
+import nitro
```

Comment:
Could you put these in `modules/python/cython/unittests`? (And hopefully they end up getting installed into `install/unittests/cython`) I have a script that picks up everything in `install/unittests` and automatically runs them.

Reply:
Wait, it looks like the caller has to manually check the file got created and clean up after, so in this case it wouldn't belong in `unittests`